test/e2e/specs/editor/blocks/links.spec.js

```diff
@@ -265,11 +265,13 @@ test.describe( 'Links', () => {
 
 		// expect settings for `Open in new tab` and `No follow`
 		await expect( page.getByLabel( 'Open in new tab' ) ).not.toBeChecked();
-		await expect( page.getByLabel( 'nofollow' ) ).not.toBeChecked();
+		await expect(
+			page.getByLabel( 'Add no-follow for search engines' )
+		).not.toBeChecked();
 
 		// Toggle both of the settings
 		await page.getByLabel( 'Open in new tab' ).click();
-		await page.getByLabel( 'nofollow' ).click();
+		await page.getByLabel( 'Add no-follow for search engines' ).click();
```

Comment:
```suggestion
		await page.getByLabel( 'nofollow' ).click();
```

This allows the wording to be agonistic and matches are loose.